### DISCUSSION TITLE
Can you add the hacktoberfest topic to this repository?

### DISCUSSION THREAD

**Discussion by kishaningithub** [pkgxdev/pkgx#843] [Ideas]
I see this has been added to some of the other repos in this org.. it would be great if that was added to this repo as well

**Comment by jhheider** [comment]
added to `pkgx`, `libpkgx`, and `brewkit`. was there another you thought might benefit?

**Reply by kishaningithub** [reply]
Nothing else as of now. Thanks for adding! :-)
